core/src/test/java/com/microsoft/applicationinsights/internal/jmx/JmxDataFetcherTest.java

```diff
@@ -107,13 +107,13 @@ private static void performTest(
     private static void verify(Map<String, Collection<Object>> result, String key, double expectedValue) {
         Collection<Object> objects = result.get(key);
         assertNotNull(objects);
-        assertEquals(objects.size(), 1);
+        assertEquals(1, objects.size());
         double value = 0.0;
         for (Object obj : objects) {
             try {
                 value += Double.parseDouble(String.valueOf(obj));
             } catch (Exception e) {
-                assertFalse(true);
```

Comment:
Please, do not do this...use `Assert.fail` if you intend to fail the test when it reaches this line.